tools/src/test/scala/scala/scalanative/NIRCompilerTest.scala

```diff
@@ -90,7 +120,6 @@ class NIRCompilerTest extends AnyFlatSpec with Matchers with Inspectors {
       """import scala.scalanative.unsafe.extern
           |
           |@extern trait Dummy {
-          |   var x: Int = extern
```

Comment:
This probably should not be removed

Reply:
Sorry, I forgot putting it back after debugging.